check-google-java-format.py

```diff
@@ -26,8 +26,10 @@
 
 try:
     from urllib import urlretrieve
+    # python 2
```

Comment:
My idea was to put this on the line where I wrote my comment, so the code would look like:

```
     from urllib import urlretrieve    # Python 2
```

In general, comments should precede rather than follow the code they describe, so if this was on a separate line, it would have gone before the import.  But I don't feel in this case it is important enough to merit its own line.